### PR TITLE
feat(backend): Added a new feature to disable automatic backups on network reset to avoid spending too much disk space

### PR DESCRIPTION
**Pull request by frol**
This will be used on production for devnet since we do heavy load there and that produces 5GB database file every day.

### CODE REVIEW

File: backend/src/sync.js [hunk 1]
```diff
@@ -428,22 +427,8 @@ async function syncGenesisState() {
   const genesisConfig = await retry(10, () =>
     nearRpc.sendJsonRpc("EXPERIMENTAL_genesis_config")
   );
-  if (
```
Comment (frol) [inline backend/src/sync.js:431]: This has been extracted in a separate routine into `index.js` file.

File: backend/src/sync.js [hunk 2]
```diff
@@ -454,7 +439,7 @@ async function syncGenesisState() {
     const genesisRecords = await retry(10, () =>
       nearRpc.sendJsonRpc("EXPERIMENTAL_genesis_records", [pagination])
     );
-    await saveGenesis(parsedGenesisTime, genesisRecords.records, offset);
+    await saveGenesis(genesisTime, genesisRecords.records, offset);
```
Comment (bowenwang1996) [inline backend/src/sync.js:442]: why is it indexed by genesis time instead of genesis height?

Comment (icerove) [reply backend/src/sync.js:442]: We need genesis time recorded and display on the frontend. We use genesis height other place to check and query blocks.

Comment (frol) [reply backend/src/sync.js:442]: It is not indexed by that. We use genesis time to indicate the creation time of Accounts.

**Pull request merged**